da/celestia/celestia.go

```diff
@@ -50,16 +51,16 @@ func WithRPCClient(rpc celtypes.CelestiaRPCClient) da.Option {
 }
 
 // WithTxPollingRetryDelay sets tx polling retry delay.
```

Comment:
incorrect godoc. should change to "WithAvailabilityCheckRetryDelay"

Reply:
fixed